### DISCUSSION TITLE
Cart Transform "merge" shows wrong bundle items in checkout

### DISCUSSION THREAD

**Discussion by abrajakabra** [Shopify/function-examples#399] [Feedback: Cart Transform API]
I'll try to explain the issue point by point.

This is meant to be read in the context of a custom bundle builder / mix and match.

User...
1. puts multiple selected products into the cart (and they get bundled by the cart transform merge function)
2. decides to remove the bundle from cart
3. adds a bundle again but **containing different items**

And here the problem begins:
In the cart context everything is still fine, but as soon as the user proceeds to the checkout page, the bundle itself shows up but with the **previously removed items**. When the user now goes back to cart, it has also updated to the **old** items.

If you add another random product into the cart before proceeding to checkout it seems to update correctly — at least it did in one of my tests (I can't verify that again right now).

**Comment by vasiliki-b** [comment]
Was this ever resolved? Experiencing the exact same issue

**Comment by Jam747** [comment]
Hi, I had this issue, 

<script>
localStorage.removeItem("__ui")
</script>


Add this to your cart liquid template.

**Comment by ewalk153** [comment]
For any of the developers experiencing this, could you post a minimum version of your cart transform code that reproduces this?

Also, what are the content of calling `/cart.js`? Do you see the 1st bundle contents or the 2nd bundle contents?
